contrib/pr/checkout.go

```diff
@@ -215,7 +215,7 @@ func main() {
 		//Use git cli command for windows
 		runCmd("git", "fetch", remoteUpstream, fmt.Sprintf("pull/%s/head:%s", pr, branch))
 	} else {
-		ref := fmt.Sprintf("refs/pull/%s/head:%s", pr, branchRef)
+		ref := fmt.Sprintf(gitea_git.PullPrefix+"%s/head:%s", pr, branchRef)
```

Comment:
Is there a reason why this isn't
```suggestion
		ref := fmt.Sprintf("%s%s/head:%s", gitea_git.PullPrefix, pr, branchRef)
```

Reply:
No specific reason other than I didn't thought of it. Your proposal is indeed more elegant.